packages/service-clients/odsp-client/src/odspClient.ts

```diff
@@ -108,7 +106,7 @@ export class OdspClient {
 		const container = await loader.resolve({ url });
 
 		// eslint-disable-next-line import/no-deprecated
-		const rootDataObject = await requestFluidObject<IRootDataObject>(container, "/");
+		const rootDataObject = (await container.getEntryPoint()) as IRootDataObject;
```

Comment:
make IRootDataObject a valid FluidObject, and use that, rather than as casting which is unsafe

Reply:
adding something like this to IRootDataObject should do it
```
	readonly IRootDataObject?: IRootDataObject
```